includes/class-salesforce.php

```diff
@@ -106,6 +110,17 @@ public static function register_api_endpoints() {
 			]
 		);
 
+		// Fetch order sync status.
+		register_rest_route(
+			self::SALESFORCE_API_NAMESPACE,
+			'/order',
+			[
+				'methods'             => \WP_REST_Server::READABLE,
+				'callback'            => [ __CLASS__, 'api_get_order_status' ],
+				'permission_callback' => '__return_true',
```

Comment:
I believe this should use the same capability applied to order edit.

Reply:
99e08ed authenticates the request and d4e3a72 updates the capability to `edit_others_shop_orders`, which should be enough to verify that the user has permissions to access the order edit screen for any order.